pkg/orchestrator/chunking.go

```diff
@@ -0,0 +1,32 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package orchestrator
+
+import "github.com/DataDog/datadog-agent/pkg/process/util"
+
+// ChunkOrchestratorPayloadsBySizeAndWeight chunks orchestrator payloads by max allowed size and max allowed weight of a chunk
+func ChunkOrchestratorPayloadsBySizeAndWeight(orchestratorPayloads []interface{}, orchestratorYaml []interface{}, weightForOrchestratorPayload func([]interface{}, int) int, maxChunkSize, maxChunkWeight int) [][]interface{} {
```

Comment:
What do you think renaming the argument `weightForOrchestratorPayload` to `weightForOrchestratorPayloadFunction` ?